CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ target_include_directories(tcmu-synthesizer
   )
 target_link_libraries(tcmu-synthesizer
   ${GLIB_LIBRARIES}
+  ${PTHREAD}
```

Comment:
It's the files libtcmu_config.c and libtcmu_log.c in libtcmu in the tcmu chunk that need the pthread addition, right? I think we want to add this under the "target_link_libraries(tcmu" line so anything using tcmu gets it automatically like is done with the other libtcmu dependencies.

Same comment for the addition below.

Reply:
Thanks, you are right. I have updated the code.